experimental/auth-logic/wrapper_interface.go

```diff
@@ -37,6 +37,10 @@ type UnattributedStatement struct {
 	Contents string
 }
 
+// This is an empty unattributed statement. It is defined as a var rather
+// than a constant because golang does not have support for const structs
+var NilUnattributedStatement = UnattributedStatement{Contents: ""}
```

Comment:
This is not particularly idiomatic in Go, it's usually preferable to just inline the empty constructor where needed.

Reply:
fixed